src/Microsoft.AspNet.Razor/Chunks/Generators/ChunkGeneratorContext.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Chunks.Generators
+{
+    public class ChunkGeneratorContext
+    {
+        protected ChunkGeneratorContext(ChunkGeneratorContext context)
```

Comment:
Is this file new or a rename of something else?

If GitHub is just confused, please add comments letting us know the previous file names (if possible).